airflow-core/src/airflow/ui/src/i18n/locales/fr/admin.json

```
@@ -0,0 +1,162 @@
+      "connectionType": "Connexion Type",
+      "host": "Hôte",
+      "port": "Port"
+    },
+    "connection_one": "Connexion",
+    "connection_other": "Connexions",
+    "delete":{
+      "deleteConnection_one": "Supprimer 1 connexion",
+      "deleteConnection_other": "Supprimer {{count}} connexions",
+      "firstConfirmMessage_one": "Vous êtes sur le point de supprimer la connexion suivante :",
+      "firstConfirmMessage_other": "Vous êtes sur le point de supprimer les connexions suivantes :",
+      "title": "Supprimer la Connexion"
+    },
+    "edit": "Modifier la Connexion",
+    "form":{
+      "connectionIdRequired": "Connexion ID est requis",
+      "connectionIdRequirement": "Connexion ID ne peut pas contenir uniquement des espaces",
+      "connectionTypeRequired": "Le type de connexion est requis",
+      "extraFields": "Champs Supplémentaires",
+      "extraFieldsJson": "Champs Supplémentaires JSON",
+      "helperText": "Le type de connexion est manquant ? Assurez-vous d'avoir installé le package Airflow Providers correspondant.",
+      "selectConnectionType": "Sélectionner le Type de Connexion",
+      "standardFields": "Champs Standards"
+    },
+    "noRowMessage": "Aucune connexion trouvée",
+    "searchPlaceholder": "Rechercher Connexions",
+    "test": "Test de Connexion",
+    "testDisabled": "Test de connexion désactivé. Veuillez contacter un administrateur pour l'activer.",
+    "typeMeta": {
+      "error": "Échec de la récupération des métadonnées du type de connexion",
+      "standardFields": {
+        "description": "Description",
+        "host": "Hôte",
+        "login": "Identifiant",
+        "password": "Mot de passe",
+        "port": "Port",
+        "url_schema": "Schéma"
+      }
+    }
+  },
+  "deleteActions":{
+    "button": "Supprimer",
+    "modal":{
+      "confirmButton": "Oui, Supprimer",
+      "secondConfirmMessage": "Cette action est irréversible.",
+      "thirdConfirmMessage": " Êtes-vous sûr de vouloir continuer ?"
+    },
+    "selected": "Sélectionné",
+    "tooltip": "Supprimer les connexions sélectionnées"
+  },
+  "formActions":{
+    "reset": "Réinitialiser",
+    "save": "Sauvegarder"
+  },
+  "plugins": {
+    "columns": {
+      "source": "Source"
+    },
+    "importError_one": "Erreur d'importation de plugin",
+    "importError_other": "Erreurs d'importation de plugins",
+    "searchPlaceholder": "Rechercher par fichier"
+  },
+  "pools": {
+    "add": "Ajouter une Pool",
+    "deferredSlotsIncluded": "Slots Différés Inclus",
+    "delete":{
+      "title": "Supprimer la Pool",
+      "warning": "Cela supprimera toutes les métadonnées liées à la pool et peut affecter les tâches utilisant cette pool."
+    },
+    "edit": "Modifier la Pool",
+    "form": {
+      "checkbox": "Check to include deferred tasks when calculating open pool slots",
+      "checkbox": "Cochez pour inclure les tâches différées lors du calcul des slots ouverts de la pool",
```

My 2 cents: "open slots" would translate to "disponibles" or "libres".
```suggestion
      "checkbox": "Cochez pour inclure les tâches différées lors du calcul des slots libres de la pool",
```

---

+1 for `libres` instead of `ouverts`

---

Done, thanks